src/Qir/Controller/QirController.csproj

```diff
@@ -1,12 +1,44 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>Microsoft.Quantum.Qir</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <BondCodegen Remove="test-artifacts\**" />
+    <BondCodegen Remove="Tests.QirController\**" />
+    <Compile Remove="test-artifacts\**" />
+    <Compile Remove="Tests.QirController\**" />
+    <EmbeddedResource Remove="test-artifacts\**" />
+    <EmbeddedResource Remove="Tests.QirController\**" />
+    <None Remove="test-artifacts\**" />
+    <None Remove="Tests.QirController\**" />
```

Comment:
Wondering if all these remove actions are needed. The one that most stands out to me is the BondCodeGen since there are no Bond schemas defined for this project.

Reply:
I didn't add any of this manually (i selected the folder and said "exclude from project" because it seemed to get added when I would run tests. Since this was automatically generated, I prefer not to change it.